documentation/docs/tutorials/openfga.mdx

```diff
@@ -0,0 +1,105 @@
+# OpenFGA and OPAL
```

Comment:
@daveads, this document needs some improvements as follows:

1. The structure of the doc isn't clear and very raw. I added below the outline we want to have for it
2. See inline comments, and preserve them in the new outline
3. Ensure you include a demo video of running OpenFGA with OPAL
4. Run Grammarly to fix English grammar and errors

Outline:
1. Intro (same as you provided)
2. Key Features (here, also expand on the distributed clients)
    1. Quick start
4. Write policies (guide, with one example, of writing OpenFGA and storing it in Git, and configure in OpenFGA)
5. Use a data fetcher (can be the http-fetcher, as we do on other tutorials) to fetch tuples to OpenFGA
6. Run OpenFGA with the provided configuration
7. Interacting with OpenFGA (write and read data, perform check) - here you should also repeat the benefit of using this APIs instead of just barebone OpenFGA instance
8. Monitoring

Reply:
alright sure...